JULIERAJ/KIDS-FIRST-Front
# Feature/10/refactor-reusable-buttons

# Feature/10/refactor-reusable-buttons

## One Line Description
Created a reusable button component based on bootstrap button with standardized styling.

## Requirements
- Custom buttons should be reusable.
- Design  of the buttons should be according to [Figma](https://www.figma.com/design/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=501-3083&t=eLLYUujZyqokFEVQ-0).

## Notes
All bootstrap buttons have been replaced with the CustomButton component. However, button CSS have only been standardized in the SignIn and Register pages.  This is because design has not been finalized for other pages. 

## Test Steps
1. Run frontend from Feature/10/refactor-reusable-buttons branch.
2. On Home page, click on "Sign Up" then:
   a.  Hover on "Sign Up" button to observe styling
   b.  Focus on "Sign Up" button  to observe styling
   c.  Manually disable "Sign Up" button in code to observe styling and immediately remove disable property
   d.  Click on "Sign Up" button to observe styling and confirm that behavior is the same.
3. Click on "Log In" link and repeat step 2 with "Log In" button.
4. Go back to Home Page and observe that "Sign Up Now", "Log In" and "Sign Up" buttons have not changed. 
   a.  Hover on buttons to observe styling
   b.  Focus on buttons to observe styling
   c.  Manually disable buttons in code to observe styling and immediately remove disable property
   d.  Click on buttons to observe styling and confirm that behavior is the same.

## UI/UX Outcome
[Link to UI/UX outcome in Sign Up and Log In pages](https://app.screencastify.com/v3/watch/4YygpsQr2jEijoGbou14)
